src/BootstrapAdminUi/package.json

```
@@ -18,15 +20,16 @@
     "jquery": "^3.5.0",
     "mark.js": "^8.11.1",
     "postcss": "^8.2.10",
-    "sass": "^1.54.8",
-    "sass-loader": "^13.0.0",
+    "sass": "~1.64.2",
+    "sass-loader": "~16.0.1",
```

I don't remember exact difference in behavior, but at vendor package usually used `^`.
```suggestion
    "sass": "^1.64.2",
    "sass-loader": "^16.0.1",
```